portfolio.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Portfolio</title>
+		<link href="normalize.css" rel="stylesheet" type="text/css">
```

Comment:
fix indentation here

Reply:
<head>
	<meta charset="UTF-8">
	<title>Portfolio</title>
	<link href="normalize.css" rel="stylesheet" type="text/css">

Like this?

Reply:
<head>
	<meta charset="UTF-8">
	<title>Portfolio</title>
	<link href="normalize.css" rel="stylesheet" type="text/css">

Like this?